task8/task8/Program.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace task8
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Введите значение аргумента функции:");
+            var x = double.Parse(Console.ReadLine());
+            Console.WriteLine($"f({x}) = {MyFunction(x)}");
+            Console.ReadKey();
+        }
+
+        static double MyFunction(double x)
+        {
+            if (x <= 0)
+                return 0;
+            else if (x > 0 && x <= 1)
+                return x;
```

Comment:
Здесь проверка `x > 0` лишняя.
```suggestion
            else if (x <= 1)
                return x;
```